health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/configs/UICustomizations.js

```
@@ -106,27 +106,39 @@ export const UICustomizations = {
+          const onActionSelect = (e) => {
+            if (e.name == "MP_ACTIONS_EDIT_SETUP") {
+              window.location.href = `/${window.contextPath}/employee/microplan/setup-microplan?key=${1}&microplanId=${row.id}&campaignId=${
+                row.campaignDetails.id
+              }`;
+            }
+            if (e.name == "MP_ACTIONS_VIEW_SUMMARY") {
+              window.location.href = `/${window.contextPath}/employee/microplan/setup-microplan?key=${10}&microplanId=${row.id}&campaignId=${
+                row.campaignDetails.id
+              }&setup-completed=true`;
+            }
+          };
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)